layouts/partials/tw/modal-sticky-anchored.html

```diff
@@ -0,0 +1,118 @@
+{{ $gaForm :=  "newsletters:papost-full" }}
```

Comment:
gaForm and gaCategory should match the .hiddenField. Maybe just default to PA Post but take .gaForm and .gaCategory if it exists.